### PR TITLE
Added WDAC exceptions creation ps HOW TO

### PR DESCRIPTION
**Pull request by jgeurten**
- Provided comprehensive PS cmds resulting in a WDAC rule exception and WDAC CI policy
- Verified the PS cmds successfully build a policy locally and converted the policy to binary
- Added description of the steps below the PS example

Open Items: 
- Do we need a topic in the WDAC docs about exceptions?

### CODE REVIEW

**Comment by jgeurten** [comment]
Can I please have @jsuther1974 and @isbrahm added as optional reviewers

File: docset/winserver2019-ps/configci/New-CIPolicy.md [hunk 1]
```diff
@@ -246,6 +246,75 @@ The first command gets drivers by using the **Get-SystemDriver** cmdlet, and the
...
+FileAttributes :
+FileException  : False
+UserMode       : True
+attributes     : {}
+
+$Rules += $Rule_1 + $Exception_1
+New-CIPolicy -MultiplePolicyFormat -FilePath ".\temp\Policy.xml" -Rules $Rules
+```
+
+The first set of commands create an allow file rule based on the CA certificate used to sign the test application as well as a deny exception rule based on the original file name of the application to block. The deny rule has a user mode and kernel mode component which requires both sections' file exception boolean fields to be set to '1'.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/configci/New-CIPolicy.md:310]:
```suggestion
The first set of commands creates an allow file rule based on the CA certificate used to sign the test application as well as a deny exception rule based on the original file name of the application to block. The deny rule has a user mode and kernel mode component which requires both sections' file exception boolean fields to be set to '1'. 
```

File: docset/winserver2019-ps/configci/New-CIPolicy.md [hunk 2]
```diff
@@ -246,6 +246,75 @@ The first command gets drivers by using the **Get-SystemDriver** cmdlet, and the
...
+UserMode       : True
+attributes     : {}
+
+$Rules += $Rule_1 + $Exception_1
+New-CIPolicy -MultiplePolicyFormat -FilePath ".\temp\Policy.xml" -Rules $Rules
+```
+
+The first set of commands create an allow file rule based on the CA certificate used to sign the test application as well as a deny exception rule based on the original file name of the application to block. The deny rule has a user mode and kernel mode component which requires both sections' file exception boolean fields to be set to '1'. 
+
+The second set of commands set the exceptions field of the allow file rule to the identifier of the exception rule. If the allow rule has both a user mode and kernel mode component, the exception fields of both components must have the identifier of the exception rule set.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/configci/New-CIPolicy.md:312]:
```suggestion
The second set of commands sets the exceptions field of the allow file rule to the identifier of the exception rule. If the allow rule has both a user mode and kernel mode component, the exception fields of both components must have the identifier of the exception rule set. 
```

File: docset/winserver2019-ps/configci/New-CIPolicy.md [hunk 3]
```diff
@@ -246,6 +246,75 @@ The first command gets drivers by using the **Get-SystemDriver** cmdlet, and the
...
+New-CIPolicy -MultiplePolicyFormat -FilePath ".\temp\Policy.xml" -Rules $Rules
+```
+
+The first set of commands create an allow file rule based on the CA certificate used to sign the test application as well as a deny exception rule based on the original file name of the application to block. The deny rule has a user mode and kernel mode component which requires both sections' file exception boolean fields to be set to '1'. 
+
+The second set of commands set the exceptions field of the allow file rule to the identifier of the exception rule. If the allow rule has both a user mode and kernel mode component, the exception fields of both components must have the identifier of the exception rule set. 
+
+The last commands merge the allow file rule and its deny rule exceptions into one rule variable which can be used in the New-CIPolicy creation step. The same process can be repeated for a deny file rule with allow exception rules.  
+
+File rule exceptions cannot use the PCA Certificate, Publisher, Signed Version or File Publisher rule levels.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/configci/New-CIPolicy.md:316]:
```suggestion
File rule exceptions cannot use the PCA Certificate, Publisher, Signed Version, or File Publisher rule levels. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by sdwheeler — CHANGES_REQUESTED** [review]
Please make the same changes to the 2022 content.

**Comment by jgeurten** [comment]
> Please make the same changes to the 2022 content.

I have made the changes to the 2022 content as well, @sdwheeler. In the future, is the expectation to make the changes to only 2022 content or both 2019, 2022? Thanks. 

Good to merge the PR from my side.

**Comment by sdwheeler** [comment]
@jgeurten You need to make changes to all versions that the change applies to. This can include the previous versions if the change is appropriate to those versions.

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**